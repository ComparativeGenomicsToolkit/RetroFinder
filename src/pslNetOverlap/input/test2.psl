415	230	0	0	0	0	0	0	+	AK124403.1-10	2281	1013	2182	chr16	88827254	1246546	1248453	20	56,16,86,12,23,6,24,11,15,11,42,16,43,7,77,27,34,92,23,24,	1013,1069,1086,1192,1205,1228,1234,1258,1273,1288,1299,1341,1360,1404,1411,1491,1520,2040,2135,2158,	1246546,1246609,1246625,1246711,1246723,1246747,1246761,1246790,1246801,1246838,1246861,1246906,1246922,1246965,1246973,1247050,1247077,1248311,1248403,1248429,
327	206	0	0	0	0	0	0	+	AB231739.1-245	621	58	621	chr11	134452384	59055955	59056547	19	93,5,78,75,6,7,10,11,32,96,11,14,4,13,7,6,4,13,48,	58,155,160,238,317,323,330,340,351,384,494,505,524,528,541,550,556,560,573,	59055955,59056048,59056054,59056133,59056208,59056226,59056238,59056258,59056281,59056313,59056409,59056422,59056436,59056450,59056464,59056471,59056478,59056483,59056499,
2068	143	0	0	0	0	0	0	-	AK124334.1-820	2221	0	2221	chr7	158821424	73871278	73874017	20	124,289,11,4,17,9,8,10,24,13,7,7,16,16,25,20,7,13,8,1583,	0,124,413,424,428,445,458,466,476,500,513,520,527,546,565,590,610,617,630,638,	73871278,73871698,73871988,73872024,73872040,73872070,73872079,73872092,73872118,73872164,73872196,73872224,73872233,73872249,73872265,73872343,73872368,73872393,73872411,73872434,
