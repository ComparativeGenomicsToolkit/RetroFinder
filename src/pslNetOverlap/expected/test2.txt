#qName	/hive/users/baertsch/retro/hg18/exp/age/netCanFam2.txt	/hive/users/baertsch/retro/hg18/exp/age/netMm9.txt	/hive/users/baertsch/retro/hg18/exp/age/netRheMac2.txt	/hive/users/baertsch/retro/hg18/exp/age/netPanTro2.txt	
AK124403.1-10	0	0	0	0	
AB231739.1-245	0	0	0	0	
